pages/ecosystem.vue

```diff
@@ -27,85 +27,97 @@
       />
-      <UiFiltersResultsLayout class="ecosystem__filters-result-section">
-        <template #filters-on-m-l-screen>
-          <UiFieldset label="Tier">
-            <client-only>
-              <bx-checkbox
-                v-for="option in tiers"
-                :key="option.name"
-                class="ecosystem__filters-result-section__tiers"
-                :checked="isTierFilterChecked(option.name)"
-                :label-text="option.name"
-                :value="option.name"
-                @bx-checkbox-changed="
-                  updateTierFilter(option.name, $event.target.checked)
-                "
-              />
-            </client-only>
-          </UiFieldset>
-        </template>
-        <template #filters-on-s-screen>
-          <UiMultiSelect
-            label="Tier"
-            :options="tiersNames"
-            :value="tierFiltersAsString"
-            @change-selection="updateTierFilters($event)"
-          />
-        </template>
-        <template #results>
-          <div class="cds--row">
-            <div
-              v-for="(member, index) in filteredMembers"
-              :key="index"
-              class="cds--col-sm-4 cds--col-xlg-8"
+      <div class="ecosystem__tiers">
+        <client-only>
+          <bx-tabs
+            trigger-content="Select an item"
+            value="Main"
+            @bx-tabs-selected="selectTab($event.target.value)"
+          >
+            <bx-tab
+              v-for="tierName in tiersNames"
+              :id="`tab${tierName}`"
+              :key="tierName"
+              :target="`panel${tierName}`"
+              :value="`${tierName}`"
             >
-              <UiCard
-                class="project-card"
-                :title="member.name"
-                :tags="member.labels"
-                :tooltip-tags="[
-                  {
-                    label: member.tier,
-                    description: getTierDescription(member.tier),
-                  },
-                ]"
-                cta-label="Go to repo"
-                :segment="{
-                  cta: `go-to-repo-${member.name}`,
-                  location: 'ecosystem-card',
-                }"
-                :to="member.url"
-              >
-                <div class="cds--row">
-                  <p class="project-card__license">
-                    {{ member.licence }}
-                  </p>
-                  <div class="project-card__star">
-                    <StarFilled16 />
-                    <p class="project-card__star-val">
-                      {{ member.stars }}
+              {{ tierName }}
+            </bx-tab>
+          </bx-tabs>
+          <div
+            v-for="tierName in tiersNames"
+            :id="`panel${tierName}`"
+            :key="tierName"
+            class="ecosystem__tier-panel"
+            role="tabpanel"
+            :aria-labelledby="`tab${tierName}`"
+          >
+            <bx-data-table page-size="2" start="0">
```

Comment:
@y4izus as mentioned in our discussion, we can remove the `bx-data-table` and simply go with [the Search component](https://web-components.carbondesignsystem.com/?path=/story/components-search--default&knob-Direction%20(direction%20in%20%3Cbx-overflow-menu-body%3E)=bottom&knob-Helper%20text%20(helper-text)=Optional%20helper%20text&knob-Label%20text%20(label-text)=Dropdown%20title&knob-Overflow%20menu%20size%20(size)=xl&knob-The%20default%20content%20of%20the%20trigger%20button%20(trigger-content)=Select%20an%20item)

```vue
<bx-search
  class="ecosystem__search"
  placeholder="Search using keywords like algorithms, simulator, or machine learning"
  @bx-search-input="searchOnMembers($event.detail.value)"
/>
```

In this example above, I added the class of `ecosystem__search` so we can override the `gray` colors, with our own `cool gray` colors. Not sure if there's a better way, but this will work, lower in the same file:

```scss
<style lang="scss">
.ecosystem {
  &__search {
    --cds-field-01: #{carbon.$cool-gray-10};
    --cds-field-04: #{carbon.$cool-gray-30};
  }
}
</style>
``` 

(just an example)

Reply:
you mean outside the tier tabs too?

Reply:
Done:
- 300b06efb8074113ee7154a95a6425f327a074e0 : Updates component and styles
- db153287a848752f8c85544586450fbb7e7390b5 : Move the search bar outside the tier panels